CODEOWNERS

```diff
@@ -42,3 +42,5 @@
 /Workbooks/SapMonitor/** @tniek @microsoft/applicationinsights-devtools
 
 /Workbooks/Virtual*Machine*/** @andrewki-msft @nasundar
+
+/Workbooks/UsageMetrics/** @MSKaiLian
```

Comment:
you need to have at LEAST 2 people here, otherwise nobody will be able to approve your things, since you'd be the only owner and you can't approve your own things.  worst case, add our team (`@microsoft/applicationinsights-devtools`) as a secondary

Reply:
add @microsoft/applicationinsights-devtools team